bower.json

```diff
@@ -11,7 +11,6 @@
     "spinner",
     "loading"
   ],
-  "private": true,
```

Comment:
Please leave this in. It means that the element is not publishable on the bower catalog :)